README.md

```diff
@@ -96,7 +97,34 @@ Connect to the container (with KUBECONFIG exported)
 make connect
 ```
 
-Accessing OKD UI
+To access the OKD UI from the host running `docker`, remember to export `OKD_CONSOLE_PORT=443` before `make cluster-up`.
+You should find out the IP address of the OKD docker container
 ```
-TODO - in the process of working out the details here.
+clusterip=$(docker inspect $(docker ps | grep "kubevirtci/$KUBEVIRT_PROVIDER" | awk '{print $1}') | jq '.[0].NetworkSettings.IPAddress' | tr -d '"' )
 ```
+and make it known in `/etc/hosts` via
+```
+cat << EOF >> /etc/hosts
+$clusterip console-openshift-console.apps.test-1.tt.testing
+$clusterip oauth-openshift.apps.test-1.tt.testing
+EOF
+```
+Now you can browse to https://console-openshift-console.apps.test-1.tt.testing
+and log in by picking the `htpasswd_provider` option. The credentials are `admin/admin`.
+
+
+To access the OKD UI from a remote client, forward incoming port 433 into the OKD cluster
+on the host running kubevirtci:
+```
+$ nic=em1  # the interface facing your remote client
```

Comment:
In order to be more user-friendly, I would recommend to extract the NIC name and its IP address programmatically:
```
nic=`ip route get 8.8.8.8 | grep via | awk '{print $5}'`
kubevirtci_ip=`ip route get 8.8.8.8 | grep via | awk '{print $7}'`
```

Reply:
> nic=`ip route get 8.8.8.8 | grep via | awk '{print $5}'`
> kubevirtci_ip=`ip route get 8.8.8.8 | grep via | awk '{print $7}'`

I don't know... These might be wrong for multi-homed hosts, where the route to the client machine is different from the route to `8.8.8.8`. I'd prefer to discuss this in a follow-up pr.

Reply:
so instead of 8.8.8.8 we can put $client_ip (which should be defined earlier).
but nvm.